aten/src/ATen/function_wrapper.py

```diff
@@ -300,8 +454,10 @@ def to_return_type(arg, option):
 
 
 def create_generic(top_env, declarations):
+    # type: (TopEnvironment, List[FunctionOption]) -> List[OrderedDict]
```

Comment:
Looks like we lost some type information here (if OrderedDict is what I think it is)

Reply:
Yes, `OrderedDict` is a dictionary that supports `pop_item`. I can't really type it as a `TypedDict` because of the extra `popitem` method, and mypy doesn't have an `OrderedTypedDict` at the moment. I could try subclassing TypedDict though and see if that gets anywhere... or refactor the code so that we don't need an OrderedDict anymore